src/components/StoreDisplay/ReviewList/ReviewList.tsx

```diff
@@ -40,18 +45,24 @@ const ReviewList: React.FC<ReviewType> = ({
   }
 
   const deleteRevieHandler = () => {
+    window.confirm('리뷰를 삭제 하시겠습니까?')
     if (storeId) {
       dispatch(deleteReview({ storeId, reviewId })).then(() => {
         dispatch(fetchAllReviews(storeId))
       })
     }
   }
 
+  const editHandler = () => {
+    dispatch(selectReview(reviewId))
+    navigate(`edit/${reviewId}`)

```

Comment:
(p5) 이런 path 는 컴포넌트에서 오타내기 딱 좋은 부분인 것 같습니다. path 를 관리하는 별도의 파일이 있어서, 생성까지 같이 해주면 좋을 것 같네요
```suggestion
    navigate(URLUtil.getReviewEditUrl(reviewId))
```